e2e/pages/caseView.page.js

```diff
@@ -136,14 +136,14 @@ module.exports = {
 
   async checkTasksDoesNotContainError(error) {
     const errors = await this.getTaskListErrors();
-
-    assert.strictEqual(errors.includes(error), false);
+    if (errors.length != 0)
```

Comment:
Not sure why we need this, would errors.includes(...) not return false if the list had length 0?